apps/console/src/features/oidc-scopes/pages/oidc-scopes.tsx

```diff
@@ -164,11 +169,15 @@ const OIDCScopesPage: FunctionComponent<OIDCScopesPageInterface> = (
     /**
     * Handle sort strategy change.
     *
-    * @param {React.SyntheticEvent<HTMLElement>} event.
-    * @param {DropdownProps} data.
+    * @param event - Click event.
+    * @param data - Dropdown data.
     */
     const handleSortStrategyChange = (event: React.SyntheticEvent<HTMLElement>, data: DropdownProps) => {
-        setSortByStrategy(SORT_BY.filter(option => option.value === data.value)[ 0 ]);
+        setSortByStrategy(SORT_BY.filter((option: {
+            key: number;
+            text: string;
+            value: string;
+        }) => option.value === data.value)[ 0 ]);
```

Comment:
Shall we can unify this type and `SORT_BY` type?

Reply:
Fixed with - https://github.com/wso2/identity-apps/pull/3649/commits/6e960be73d4ef5b3f0fe522eb5ba171bfef6f91f